pxr/usd/usdLux/lightAPI.h

```diff
@@ -267,7 +301,22 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // INTENSITY 
     // --------------------------------------------------------------------- //
-    /// Scales the power of the light linearly.
+    /// Scales the brightness of the light linearly.
+    /// 
+    /// Expresses the "base", unmultiplied luminance emitted (L) of the light,
+    /// in nits (cd∕m²):
+    /// 
+    /// <center><b>
+    /// L<sub>Scalar</sub> = intensity
+    /// </b></center>
+    /// 
+    /// More precisely, the lights' emission is in units of spectral radiance
```

Comment:
"More precisely" as used throughout this document slightly carries the meaning that there might be "even more precise" ways to describe things, but that's not the case. Can we use a wording such as "Normatively" or "Mathematically" instead? I don't want people to wonder if there is another shoe to drop in the future.

Reply:
Updated all these to "Normatively" or "Mathematically" as appropriate:
https://github.com/PixarAnimationStudios/OpenUSD/pull/3182/commits/a1d7d8328149bf6067832d65c79f003fcfc262fa#diff-11634bb5438749e3c529c9f44011d464e79bdb1352c0b7fc7c9408bece4415e4R314